src/app/modules/main/chat_section/chat_content/users/module.nim

```diff
@@ -33,17 +38,63 @@ method delete*(self: Module) =
   self.controller.delete
 
 method load*(self: Module) =
-  singletonInstance.engine.setRootContextProperty("usersModule", self.viewVariant)
-  
   self.controller.init()
   self.view.load()
 
 method isLoaded*(self: Module): bool =
   return self.moduleLoaded
 
 method viewDidLoad*(self: Module) =
+  # add me as the first user to the list
+  let loggedInUserDisplayName = singletonInstance.userProfile.getName() & "(You)"
```

Comment:
We might need a TODO to have this translated one day

Reply:
@jrainville yes, that's true, we have the same situation in `app-search` module for search result messages. We need to see how we should do a translation on the backend side since there will be more needs for that in the future, not just these two places.